features/step_definitions/homepage_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given('I open the Rozetka website') do
+  @homepage = Web::Pages::HomePage.new
+  page.current_window.maximize
+  visit '/'
```

Comment:
it will not work. Seems like it should be `visit 'https://rozetka.com.ua/'` or create singleton class, name it as  constants and then use it here for the url